applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -22,6 +22,12 @@
 
         @fragments.headTonal(media)
         <div class="content__main tonal__main tonal__main--@toneClass(media)">
+            @media match {
+                case video: Video => {
+                    <div class="video__fader"></div>
```

Comment:
I'd probably just have it as `video-fader` as it's an element in it's own right.